debug/gdbserver.py

```diff
@@ -567,6 +567,13 @@ def test(self):
         assertEqual(123, self.gdb.p("$x1"))
         assertEqual(123, self.gdb.p("$csr832"))
 
+class NonExistentCsr(RegsTest):
+    def test(self):
+        #Print old version of $misa at 0xF10 (moved to 0x301)
+        self.gdb.p("$csr3856")
```

Comment:
Sorry, not getting it. I tried 
`output = self.gdb.p("..")` 
but that's wrong.  Maybe it's on stderr or something:

```
(gdb) p/x $csr3856
p/x $csr3856
Could not fetch register "csr3856"; remote failure reply 'E0E'
(gdb) ////////////////////////////////////////
::::::::[ ran 2 tests in 124s ]:::::::::
1 tests returned exception
    NonExistentCsr
1 tests returned pass
```